zeebe/gateway-protocol-impl/src/main/resources/gateway-service-config.json

```diff
@@ -34,7 +33,8 @@
         {"service": "gateway_protocol.Gateway", "method": "PublishMessage"},
         {"service": "gateway_protocol.Gateway", "method": "ThrowError"},
         {"service": "gateway_protocol.Gateway", "method": "UpdateJobRetries"},
-        {"service": "gateway_protocol.Gateway", "method": "UpdateJobTimeout"}
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobTimeout"},
+        {"service": "gateway_protocol.Gateway", "method": "SetVariables"}
```

Comment:
🔧 The list appears ordered alphabetically. Let's place `SetVariables` between `PublishMessage` and `ThrowError` to continue this.